arrow-refined-types/build.gradle

```diff
@@ -43,6 +40,10 @@ kotlin {
             }
         }
     }
+
+    js {
+        nodejs()
```

Comment:
should we enable the browser too or it does not matter?

Reply:
Of course, done, thanks!! :raised_hands: